src/cryptonote_protocol/cryptonote_protocol_handler.h

```diff
@@ -112,6 +113,15 @@ namespace cryptonote
     void log_connections();
     std::list<connection_info> get_connections();
     const block_queue &get_block_queue() const { return m_block_queue; }
+    const std::uint64_t max_average_of_blocksize_in_queue() {
```

Comment:
```suggestion
    std::uint64_t max_average_of_blocksize_in_queue() {
```

Warning in build log